### ISSUE TITLE
Make -m/--min-scene-len a global option

### ISSUE THREAD

**Issue by tonycpsu** [Breakthrough/PySceneDetect#131]
As we discussed in #128, there seems to be some merit to moving the `-m`/`--min-scene-len` option, which is duplicated between `detect-content` and `detect-threshold` to the main option parsing.  My interest here isn't just in reducing the duplication, though that's certainly a nice side effect, but in adding the option to drop short scenes instead of having them combined with neighbors, which seems to be the current behavior.

So here's what I'm thinking of doing after #128 is merged:

1. Remove `--min-scene-len` from `detect-content` and `detect-threshold`,
2. Add it to `scenedetect_cli`.
3. Add a new option, something like `--drop-short-scenes`, which, when `True`, has `detect-content` and `detect-threshold` ignore the `--min-scene-len` option, deferring handling of the option until after the scene list is generated.
4. Update `context.py` to drop scenes shorter than `--min-scene-len` when `--drop-short-scenes` is `True`.

Thoughts?

**Comment by Breakthrough** [comment]
Sounds good to me, if anyone else has any opinions or thoughts on the matter feel free to voice them.  Something like this has been hinted at by multiple other users at this point though, so I'm certain there is some utility in changing this.

Just one question, does this imply that when `--drop-short-scenes` is *not* specified, the default behavior is to merge the scenes instead of dropping them?

**Comment by tonycpsu** [comment]
> Just one question, does this imply that when `--drop-short-scenes` is _not_ specified, the default behavior is to merge the scenes instead of dropping them?

Yeah.  Basically, `detect-content` and `detect-threshold` will have a line like this:

```python
    min_scene_len = 0 if ctx.obj.drop_short_scenes else ctx.obj.min_scene_len```

**Comment by Breakthrough** [comment]
Resolved in v0.5.5.
